CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0] - 2018-12-13
+
+### Added
+
+- The pool boy can check if a corps is dead or alive before cleaning it from the pool.
```

Comment:
corps -> corpse

or, since a corpse is normally dead, you might call it "floating body"